src/sentry/killswitches.py

```diff
@@ -37,9 +38,19 @@ def _update_project_configs(
     # or removed
     changed_project_ids = old_project_ids ^ new_project_ids
 
-    with click.progressbar(changed_project_ids) as ids:
-        for project_id in ids:
-            if project_id is not None:
+    if None in changed_project_ids:
+        with click.progressbar(length=Organization.objects.count()) as bar:
+            # Since all other invalidations, which would happen anyway, will de-duplicate
+            # with these ones the extra load of this is reasonable.  A temporary backlog in
+            # the relay_config_bulk queueu is just fine.
+            for org in Organization.objects.all():
+                schedule_invalidate_project_config(
+                    trigger="invalidate-all", organization_id=org.id, countdown=0
+                )
+                bar.update(1)
```

Comment:
I basically inlined the `invalidate_all_project_configs` task here: it allows for a progress bar and explicitly shows when all the tasks have been scheduled rather than having to guess based on what you can see on the queue.  I think I prefer this slightly over spawning the task to do this so would probably delete the task before merging this.

Reply:
i deleted the task, this is better i think